x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -37,6 +37,23 @@ async function deleteIndices(indices: string[], client: Client) {
   }
 }
 
+async function addUser(esClient: Client) {
+  try {
+    await esClient.transport.request({
```

Comment:
I'm not sure about what I'm going to ask but, is there a way to create an user using a kibana service or something? I'm just wondering using esClient directly will generate issues in the future if the way a user is created changes or something.

Reply:
I'll see what I can find.

Reply:
So I looked, but can only find kibana [APIs for managing roles](https://www.elastic.co/guide/en/kibana/master/role-management-api.html) and not users. Will have to use the [Elastic API](https://www.elastic.co/guide/en/elasticsearch/reference/master/security-api-put-user.html) for creating and updating users for now.